File: crates/target_csharp_system_text/output/description/RootDiscriminatorWithDescriptionBar.cs
using System.Text.Json.Serialization;

namespace JtdCodegenE2E
{
    /// <summary>
    /// A description for discriminator variant
    /// </summary>
    public class RootDiscriminatorWithDescriptionBar : RootDiscriminatorWithDescription
    {
        [JsonPropertyName("foo")]
        public string Foo { get => "bar"; }
    }
}

File: crates/target_java_jackson/output/description/RootDiscriminatorWithDescriptionBar.java
package com.example;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;

/**
 * A description for discriminator variant
 */
@JsonSerialize
public class RootDiscriminatorWithDescriptionBar extends RootDiscriminatorWithDescription {
    public RootDiscriminatorWithDescriptionBar() {
    }
}
